### PR TITLE
Add instructions for Mac OS X.

### PR DESCRIPTION
**Pull request by jeremymeng**
- Fix the incorrect port.
- Add instructions for Mac OS X.

### CODE REVIEW

**Comment by jeremymeng** [comment]
@twsouthwick Can you please take a look! Thanks!

File: docs/GettingStartedOSX.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+#Getting Started for Mac OS X#
```
Comment (twsouthwick) [inline docs/GettingStartedOSX.md]: Looks good, but I'm worried that a lot of this is duplicated from the Linux How-To so we would end up having to ensure things are in sync between the two.  What do you think?

Comment (jeremymeng) [inline docs/GettingStartedOSX.md]: Yes I thought about that too.  I could merged the two into one, but I was worried about it getting messy, with two different paths to choose.  Let me try that and see how it looks.

File: docs/GettingStartedLinuxOSX.md [hunk 2]
```diff
@@ -84,9 +108,9 @@ Run the app by
     ~/site/Kestrel
 ```
 
-Use your favorite web browser to navigate to `http://localhost:5004`.
+Use your favorite web browser to navigate to `http://localhost:5001`.
 `lynx` and `links` are text-based web browsers, if you don't have GUI
-installed for your Linux machine.
+ginstalled for your Linux machine.
```
Comment (twsouthwick) [inline docs/GettingStartedLinuxOSX.md]: spelling

File: docs/GettingStartedLinuxOSX.md [hunk 3]
```diff
@@ -1,16 +1,16 @@
-#Getting Started for Linux#
+#Getting Started for Linux/OS X#
```
Comment (twsouthwick) [inline docs/GettingStartedLinuxOSX.md]: Can you update the link from README so it's clear that this also works for OSX?

File: docs/GettingStartedLinuxOSX.md (hunk 2 above)
Comment (jeremymeng) [inline docs/GettingStartedLinuxOSX.md]: Fixed.

File: docs/GettingStartedLinuxOSX.md (hunk 3 above)
Comment (jeremymeng) [inline docs/GettingStartedLinuxOSX.md]: Yes. Added.

**Comment by twsouthwick** [comment]
Great! LGTM

**Pull request merged**